config-sets/config04/apply-config.sh

```diff
@@ -0,0 +1,133 @@
+#include the path of the IS Pack for the productHome variable when trying locally
+#################################################
+#Examples
+#productHome=/home/shavantha/products/wso2is/550/wso2is-5.5.0
+#configSetHome=/home/shavantha/products/jmeter/apache-jmeter-3.3/bin/isscripts/shavantha/identity-test-integration/config-sets/config04
+##################################################
+
+productHome=$productHome
+configSetHome=$configSetHome
+folderPath=/repository/conf/
+webappPath=/repository/deployment/server/webapps/
+
+
+serverHost=$serverHost
+serverPort=$serverPort
+#Setup specific host details
+ISServerHost=$ISServerHost
+ISServerPort=$ISServerPort
+TomcatServerHost=$TomcatServerHost
+TomcatServerPort=$TomcatServerPort
```

Comment:
We can reuse the default variable serverHost, serverPort, tomcatHost, tomcatPort instead of defining new variables